archetypes/gwtp-appengine-guice/pom.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.arcbees.archetypes</groupId>
+    <artifactId>gwtp-appengine-guice-archetype</artifactId>
+    <version>1.1-SNAPSHOT</version>
+    <packaging>maven-archetype</packaging>
+
+    <name>gwtp-appengine-guice-archetype</name>
+    <description>GWTP Appengine Guice application</description>
+
+    <properties>
+        <archetype.version>2.3</archetype.version>
+        <maven-resources-plugin.version>2.7</maven-resources-plugin.version>
+    </properties>
+
+    <build>
+        <extensions>
+            <extension>
+                <groupId>org.apache.maven.archetype</groupId>
+                <artifactId>archetype-packaging</artifactId>
+                <version>${archetype.version}</version>
+            </extension>
+        </extensions>
+
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <artifactId>maven-archetype-plugin</artifactId>
+                    <version>${archetype.version}</version>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-resources-plugin</artifactId>
+                <version>${maven-resources-plugin.version}</version>
+                <executions>
+                    <execution>
+                        <id>copy-resources</id>
+                        <phase>prepare-package</phase>
+                        <goals>
+                            <goal>copy-resources</goal>
+                        </goals>
+                        <configuration>
+                            <outputDirectory>${basedir}/target/classes/archetype-resources</outputDirectory>
+                            <resources>
+                                <resource>
+                                    <directory>${basedir}/../gwtp-basic/src/main/resources/archetype-resources</directory>
```

Comment:
I thought this could be easier since we won't have to maintain 2 gwtp-basic with this. It copies the gwtp-basic archetype into the target folder, since it's only client files and this one are only server files